### PR TITLE
refactor(objectviewer): connect it

### PR DESCRIPTION
**Pull request by jmfrancois**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

The objectviewer declares it's own state.

**What is the new behavior?**

The objectviewer is connected using cmfConnect

**Does this PR introduce a breaking change?**

- [x] No

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://289.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://289.talend.surge.sh)

File: packages/containers/src/ObjectViewer/ObjectViewer.test.js [hunk 1]
```diff
@@ -1,14 +1,10 @@
 import React from 'react';
-import renderer from 'react-test-renderer';
 import { Map } from 'immutable';
```
Comment (sdwvit) [inline packages/containers/src/ObjectViewer/ObjectViewer.test.js]: eslint says its not used, pls check

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://289.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://289.talend.surge.sh)

**Pull request merged**